### PR TITLE
Feature 34 input field calendar

### PR DESCRIPTION
**Pull request by bajserowiczMikolaj**
Imported calendar feature with npm Calendar (https://www.npmjs.com/package/react-calendar). and date peaker npm (https://www.npmjs.com/package/react-time-picker). Page styled according to Figma and tested.

### CODE REVIEW

**Comment by tomas-trls** [comment]
Also rewrote the test for the data card since they were not passing anymore once we updated the data types

File: src/containers/UserInfoContainer/UserInfoContainer.scss [hunk 1]
```diff
@@ -1,52 +1,166 @@
...
+          font-size: 1.2rem;
+          font-weight: 800;
+          color: $color-white;
+        }
+      }
+    }
   }
 }
 
 @media screen and (min-width: 768px) {
```
Comment (TristanPere) [inline src/containers/UserInfoContainer/UserInfoContainer.scss:117]: use variable tablet breakpoint

File: src/pages/Login/Login.scss [hunk 2]
```diff
@@ -94,30 +94,28 @@
...
       }
-      25% {
-        opacity: 1;
-      }
-      90% {
+      to {
+        top: 150px;
         opacity: 1;
       }
     }
```
Comment (TristanPere) [inline src/pages/Login/Login.scss:121]: I understand this is something you worked on previously and saw space for improvement, however try to stick to the pages you are working on to make sure there is no cross editing of areas and notify the PO when refactoring work from a previous ticket for reasons stated.

File: src/styles/base/_typography.scss [hunk 3]
```diff
@@ -1,6 +1,6 @@
 @use "../abstracts/variables" as *;
 
-@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Jost:wght@700&family=Lato:ital@1&family=Mulish:wght@600&family=Poppins:wght@400;500;600;700&family=Space+Mono:wght@700&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&family=Jost:wght@700&family=Lato:wght@100;300;400;700;900&family=Mulish:wght@600&family=Poppins:wght@400;500;600;700&family=Space+Mono:wght@700&display=swap");
```
Comment (TristanPere) [inline src/styles/base/_typography.scss:3]: what was this change for?

File: src/containers/UserInfoContainer/UserInfoContainer.jsx [hunk 4]
```diff
@@ -1,11 +1,17 @@
...
+import DatePicker from "react-date-picker";
 import Button from "../../components/Button/Button";
-import Calendar from "../../assets/images/appointments-temporary/Calendar.png";
+import DropdownField from "../../components/DropdownField/DropdownField";
+import InputField from "../../components/InputField/InputField";
+
+import "./UserInfoContainer.scss";
 
 const UserInfoContainer = () => {
+  //You can use the day from the calendar with this useState
```
Comment (TristanPere) [inline src/containers/UserInfoContainer/UserInfoContainer.jsx:12]: remove comment

**Review by TristanPere — CHANGES_REQUESTED** [review]
Looking really solid the calendar looks amazing plus the mobile responsiveness is above expectations. Code and testing looks clean. Please could you change the file names of UserInfoContainer to BookingForm or something of the type as the file name doesn't accurately describe the functionality of the file.

File: src/styles/base/_typography.scss (hunk 3 above)
Comment (tomas-trls) [reply src/styles/base/_typography.scss:3]: We changed the font family fron Italic to normal, you can notice in the figma, it's using the normal font where in the link. The part &family=Lato:ital only gets fonts with italic properties. We needed normal fonts

**Review by TristanPere — APPROVED** [review]

**Pull request merged**